clients/drcachesim/CMakeLists.txt

```diff
@@ -624,7 +624,8 @@ if (BUILD_TESTS)
       "${PROJECT_SOURCE_DIR}/clients/drcachesim/tests/drmemtrace.threadsig.x64.tracedir")
     set(tmp_output_dir ${PROJECT_BINARY_DIR}/record_filter_tests_tmp_output)
     file(MAKE_DIRECTORY ${tmp_output_dir})
-    add_executable(tool.drcachesim.record_filter_unit_tests tests/record_filter_unit_tests.cpp)
+    add_executable(tool.drcachesim.record_filter_unit_tests
```

Comment:
Maybe the name should be s/drcachesim/drcacheoff/ as that is how we name all our offline tests

Reply:
Done.